### ISSUE TITLE
Hyprpanel doesnt update to matugen's colour

### ISSUE THREAD

**Issue by NaiveInvestigator** [Jas-SinghFSU/HyprPanel#535]
**Describe the bug**
When changing the wallpaper, hyprpanel doesn't pick up matugen's colour
Jas narrowed down to the printing "ok" bit breaks the json parsing.

**To Reproduce**
run `matugen image PATH/TO/WALLPAPER -t scheme-content --contrast 0 --json hex`

if you get an "ok" after the json you have this issue too.

**The Fix**
edit your `~/.config/ags/services/matugen/index.ts` and go to line 36
around there find the relevant code section and change it to this
```
        let contents = await bash(
            `matugen image ${wallpaperPath} -t scheme-${scheme_type.value} --contrast ${normalizedContrast} --json hex`,
        );

        contents = contents.match(/{.*}/s)?.[0] || '';
```
basically change the const to let and add the extra regex bit

**Comment by NaiveInvestigator** [comment]
ok it seems i have finally narrrowed down the issue

I get that "ok" message due to my matugen config

to be specific this part:

```
[templates.hyprland]
input_path = './templates/hyprland-colors.conf'
output_path = '~/.config/hypr/colors.conf'
post_hook = 'hyprctl reload'

```

when i run `hyprctl reload`, after reloading everything it prints out "ok", which seems to cause the issue

My matugen config is directly taken from https://github.com/InioX/matugen-themes not some weird edge case.

**Comment by Jas-SinghFSU** [comment]
fixed in #588
